components/SpeakersSection.js

```diff
@@ -22,12 +22,32 @@ export default function TicketsSection () {
   return (
     <div className='speakers-section'>
       <SectionTitle>Speakers</SectionTitle>
-      <ul>
-        {speakers.map((speaker, index) => <li>
-          <Speaker {...speaker} />
-        </li>)}
-      </ul>
+      <div className='container'>
+        <ul className='row'>
+          {speakers.map((speaker, index) => <li className="col-sm-4">
+            <Speaker {...speaker} />
+          </li>)}
+        </ul>
+      </div>
       <style jsx>{`
+        .container {
+          width: 1060px;
```

Comment:
Between 991~1060 pixels, the speaker section overflows the layout.

Please change this to `max-width`.